src/markdown-pages/automate-workflows/get-started-terraform.mdx

```diff
@@ -37,273 +45,327 @@ To use this guide, you should have some basic knowledge of both New Relic and Te
-
-</Step>
-<Step>
+```hcl
+provider "newrelic" {
+  account_id = 12345   # Your New Relic account ID
+  api_key = "NRAK-***" # Your New Relic Personal API Key
+  region = "US"        # US or EU (defaults to US)
+}
+```
 
-## Add a data source
+With your New Relic provider configured, we can initialize Terraform by running the `terraform init` command. This command will result in the New Relic provider being installed and registered with Terraform.
 
-The alert policy you defined in `main.tf` does not contain any alert condition. You are going to add an alert condition linked to your application.
+```bash
+terraform init
+```
 
-To store your application's information, you need to add a [data source](https://www.terraform.io/docs/configuration-0-11/data-sources.html).
+<Tip>
+<small>You can also configure the New Relic provider using <a target="_blank" href="https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/guides/provider_configuration">environment variables</a>. This can be useful for setting default values for your provider configuration.
+</small><br /><br />
 
-```hcl lineNumbers=false
-provider "newrelic" {}
+<small>For more information about configuring the New Relic provider, please feel free to check out our official provider <a target="_blank" href="https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/guides/provider_configuration">documentation</a>.</small>
+</Tip>
 
-# Data Source
-data "newrelic_entity" "app_name" {
-  name    = "my-app" # This must be an exact match of your app name in New Relic (case sensitive)
-  domain  = "APM"
-  type    = "APPLICATION"
-}
+</Step>
 
-resource "newrelic_alert_policy" "alert_policy_name" {
-  name = "My Alert Policy Name"
-}
-```
+<!-- ALERT POLICY -->
 
-</Step>
 <Step>
 
-## Add an alert condition to the alert policy
+## Create a New Relic alert policy with the golden signal alerts
 
-Now you can add the alert condition. For example, you can create a `critical` alert condition that is triggered when the [Apdex](https://docs.newrelic.com/docs/apm/new-relic-apm/apdex/apdex-measure-user-satisfaction) of your application falls below `0.75` for five minutes.
+With the New Relic provider configured, we can define an alerting strategy for your application that's already reporting to your New Relic account.
 
-```hcl lineNumbers=false
-provider "newrelic" {}
+Since we're targeting a specific application, we'll configure a [`newrelic_entity`](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/data-sources/entity) data source which will fetch the application information from New Relic and allow us to reference that data elsewhere in the configuration.
 
-data "newrelic_entity" "app_name" {
-  name    = "my-app"
-  domain  = "APM"
-  type    = "APPLICATION"
+```hcl
+data "newrelic_entity" "example_app" {
+  name = "Your App Name" # Must be an exact match to your application name in New Relic
+  domain = "APM"
```

Comment:
Good question. In general I think it's fair to assume that most customers start with an application within the APM domain which is our flagship product. That being said, you've asked a fair question and others likely will as well. Maybe a "tip" callout that helps explain the domains could help provide context and link out to the domain docs. Thoughts on this copy below?

> Note: The available domain categories are `APM`, `BROWSER`, `INFRA`, `MOBILE`, and `SYNTH`. You will want to adjust your associated configuration accordingly to ensure your entity is found. For more information please see _docs_...

Please feel free to wordsmith and also open to other suggestions 🙂

Reply:
I think this works, and the copy is good! May I also offer another potential solution?

```hcl
data "newrelic_entity" "example_app" {
  domain = "APM"  # or BROWSER, INFRA, MOBILE, SYNTH, depending on your entity's domain
}
```

This is kind of nice because we don't have to worry about adding text between the code block and the callout and it also gets the info close to the code itself. That said, I'll leave the ultimate choice to you!